README.md

```diff
@@ -34,17 +23,19 @@ point of the sample React app is [`src/ReactApp.re`](src/ReactApp.re).
 
 ## Commands
 
-In opam / dune projects, our personal preference is to group commonly used commands in a Makefile. This is completely optional.
+All the build commands are defined in the `package.json` `scripts` field. This
+is completely optional, and other tools like `make` could be used.
 
-You can see all available commands by running `make help` or just `make`. Here
-are a few of the most useful ones:
+You can see all available commands by running `npm run`. There are explanations
```

Comment:
I see that running `npm run` only shows the script name and the corresponding commands. I guess `scriptsComments` is just a convention, and is totally ignored by `npm run`?

Reply:
Yeah `scriptsComments` is just a hacky way to somehow document these commands. As they also appear in the README, are listed in `npm run`, and then somehow documented with this `scriptsComments` hack, I hope that with all this we can somehow replace the old `make help` behavior 😄